spec/gradebook_spec.rb

```diff
@@ -0,0 +1,40 @@
+require './lib/gradebook'
+require './lib/course'
+require './lib/student'
+
+RSpec.describe GradeBook do
+  describe '#initialize' do
+    it 'exists' do
+      gradebook = GradeBook.new("Prof. Jerome")
+    
+      expect(gradebook).to be_an_instance_of(GradeBook)
+      expect(gradebook.courses).to eq([])
+    end
+  end
+
+  describe '#data' do
+    it 'adds data' do
+      gradebook = GradeBook.new("Prof. Jerome")
+      course = Course.new("Calculus", 2)
+       
+      gradebook.add_course(course.name)
+
+      expect(gradebook.courses).to eq(["Calculus"])
+    end
+
+    it 'adds students' do
+      gradebook = GradeBook.new("Prof. Jerome")
+      course = Course.new("Calculus", 2)
+      student1 = Student.new({name: "Morgan", age: 21})
+      student2 = Student.new({name: "Jordan", age: 29})
+
+      course.enroll(student1)
+      course.enroll(student2)
+      gradebook.list_all_students
+
+      expect(gradebook.list_all_students).to be_a Hash
+       
+
```

Comment:
Would like to see more tests here such as
```suggestion
      expect(gradebook.list_all_students).to be_a Hash
       
      expect(gradebook.list_all_students).to eq({course => [student1, student2]})
```